src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -128,6 +141,22 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void CheckForUnsupportedArgs(Execution::Context& context)
+    {
+        const auto& unsupportedArgs = context.Get<Execution::Data::Installer>().value().UnsupportedArguments;
+
+        if (!unsupportedArgs.empty())
+        {
+            for (auto unsupportedArg : unsupportedArgs)
+            {
+                if (context.Args.Contains(GetUnsupportedArgumentType(unsupportedArg)))
+                {
+                    context.Reporter.Error() << Resource::String::UnsupportedArgument << " --" << UnsupportedArgumentToString(unsupportedArg) << std::endl;

```

Comment:
Nit: I'd make this a warning instead of an error

```suggestion
                    context.Reporter.Warn() << Resource::String::UnsupportedArgument << " --"_liv << UnsupportedArgumentToString(unsupportedArg) << std::endl;
```

Reply:
Changed to suggested